src/test/cdk/util.ts

```diff
@@ -0,0 +1,26 @@
+/*!
+ * Copyright 2018-2019 Amazon.com, Inc. or its affiliates. All Rights Reserved.
+ * SPDX-License-Identifier: Apache-2.0
+ */
+
+import * as path from 'path'
+import { Uri, WorkspaceFolder } from 'vscode'
+import { makeTemporaryToolkitFolder } from '../../shared/filesystemUtilities'
+
+export async function createWorkspaceFolder(
+    prefix: string
+): Promise<{
+    workspacePath: string
```

Comment:
workspacePath can be derived from workspaceFolder, no need to return both

Reply:
removed workspacePath